docs/admin/support.rst

```diff
@@ -2,10 +2,10 @@ Getting support for Weblate
 ===========================
 
 
-Weblate is copylefted libre software with community support.
-Subscribers receive priority support at no extra charge. Prepaid help packages are
-available for everyone. You can find more info about current support
-offerings at <https://weblate.org/support/>.
+*Weblate* is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge.
+Prepaid help packages are available for everyone.
+Find info about current support offerings at https://weblate.org/support.
```

Comment:
There was some escaping for URLs ending in dots in another PR. Maybe needed here?